hermes-consumers/src/main/java/pl/allegro/tech/hermes/consumers/config/ConsumerConfiguration.java

```diff
@@ -161,6 +161,11 @@ public HttpClient http1Client(HttpClientsFactory httpClientsFactory) {
         return httpClientsFactory.createClientForHttp1("jetty-http-client");
```

Comment:
Maybe we should change the previous client name to the `jetty-http-serial-client` to stay consistent ?

Reply:
fixed